### PR TITLE
Plumb custom image

### PR DESCRIPTION
**Pull request by pweil-**
Builds on https://github.com/kargakis/acs-engine/pull/16

Working but without data disks on the VMs.  

@kwoodson @jim-minter @kargakis

### CODE REVIEW

File: parts/openshift/3.7/bastion.json [hunk 1]
```diff
@@ -107,6 +107,7 @@
   },
   "variables" : {
     "apiVersion" : "2015-06-15",
+    "apiVersionVM" : "2016-04-30-preview",
```
Comment (kargakis) [inline parts/openshift/3.7/bastion.json]: Let's use `2017-03-30` 

> It is recommended to use an API version later than 2016-04-30-preview as there were breaking changes between 2016-04-30-preview and 2017-03-30.

https://docs.microsoft.com/en-us/azure/virtual-machines/windows/using-managed-disks-template-deployments#managed-disks-template-formatting

File: parts/openshift/resources.t [hunk 2]
```diff
@@ -383,14 +383,28 @@
       "name" : "masteravailabilityset",
       "location" : "[variables('location')]",
       "apiVersion" : "[variables('apiVersionCompute')]",
-      "properties" : {}
+      "properties": {
+                      "platformUpdateDomainCount": "[variables('updateDomains')]",
```
Comment (kargakis) [inline parts/openshift/resources.t]: Indentation seems a bit off

File: parts/openshift/vars.t [hunk 3]
```diff
@@ -1,5 +1,5 @@
-    "gituser" : "kargakis",
-    "branch" : "master",
+    "gituser" : "pweil-",
+    "branch" : "plumb-custom-image",
```
Comment (kargakis) [inline parts/openshift/vars.t]: Drop these changes

**Comment by kargakis** [comment]
Can you also drop the storage accounts while here?

**Comment by pweil-** [comment]
> Can you also drop the storage accounts while here?

Not confident yet that we don't need them but if we can get the lun1 disk attached without them then I'll rip em out.

**Comment by pweil-** [comment]
Actually, we are getting the two disks attached.  I will proceed with removing the storage accounts while @kwoodson looks at the VMs that this created

```
[root@bastion ~]# df -h
Filesystem                              Size  Used Avail Use% Mounted on
/dev/mapper/rootvg-rootvol              8.0G  2.4G  5.7G  30% /
devtmpfs                                1.7G     0  1.7G   0% /dev
tmpfs                                   1.7G     0  1.7G   0% /dev/shm
tmpfs                                   1.7G  408K  1.7G   1% /run
tmpfs                                   1.7G     0  1.7G   0% /sys/fs/cgroup
/dev/sda1                               969M  127M  777M  14% /boot
/dev/mapper/rootvg-var                   16G  698M   16G   5% /var
/dev/mapper/docker_vg-docker--root--lv  200G   33M  200G   1% /var/lib/docker
/dev/sdb1                               6.8G  2.1G  4.4G  32% /mnt/resource
tmpfs                                   344M     0  344M   0% /run/user/0
```

**Comment by kwoodson** [comment]
I verified the docker_vg-docker--root---lv was setup on the data disk:
```
[root@bastion ~]# pvs
  PV         VG        Fmt  Attr PSize    PFree
  /dev/sda2  rootvg    lvm2 a--    25.00g 1.00g
  /dev/sdc1  docker_vg lvm2 a--  <200.00g    0 
```

**Comment by pweil-** [comment]
@kargakis storage accounts for everything except registry and pvs removed, removed depends on for the master/node/infra storage accounts.

Successful run took around 10-11 minutes

File: parts/openshift/vars.t [hunk 4]
```diff
@@ -255,8 +252,10 @@
     "subscriptionId" : "[subscription().subscriptionId]",
     "StorageAccountLoggingMetricsVolumesVolumeType" : "Premium_LRS",
     "apiVersion" : "2015-06-15",
-    "apiVersionCompute" : "2015-06-15",
+    "apiVersionCompute" : "2016-04-30-preview",
```
Comment (kargakis) [inline parts/openshift/vars.t]: Update this to 2017-03-30

**Comment by kargakis** [comment]
Nevermind the bot, a bad deployment was rolled out; fixed in https://github.com/kubernetes/test-infra/pull/7253

**Pull request merged**